### ISSUE TITLE
config file options not being honored

### ISSUE THREAD

**Issue by AXington** [mintty/wsltty#46]
I added a config file to %APPDATA%\wsltty\config that has some color schemes selected, however they're not being loaded at all. The colors are still the default colors.

The command I'm using to run it is ```%LOCALAPPDATA%\wsltty\bin\mintty.exe --wsl --configdir="%APPDATA%\wsltty" -o Locale=C -o Charset=UTF-8 /bin/wslbridge -C~ -t /bin/bash```

**Comment by mintty** [comment]
Please describe your configuration setup more precisely. What's the contents of your %APPDATA%\wsltty\config file? Where did you place the colour scheme files?

**Comment by AXington** [comment]
The docs imply that the colors should just be put directly into %APPDATA%\wsltty\config. 
```
file config in directory given with mintty option --configdir
This is %APPDATA%\wsltty\config in the default wsltty installation
```
In fact, the docs aren't very good at explaining the setup of the config file, but that's for a different conversation. The current file has the output of the https://ciembor.github.io/4bit/ site copied and pasted into it. I created the file in vim, then copied it over. I actually tried it in both %APPDATA%\wsltty and %LOCALAPPDATA%\wsltty as there is no wsltty in %APPDATA% by default. %APPDATA% being /mnt/c/Users/my_user/AppData and %LOCALAPPDATA% being /mnt/c/Users/my_user/AppData/Local.

The file currently looks like this:
```
 cat /mnt/c/Users/Andrew/AppData/wsltty/config
BackgroundColour=0,0,0
ForegroundColour=255,255,255
CursorColour=255,255,255
Black=0,0,0
BoldBlack=38,38,38
Red=184,123,123
BoldRed=224,197,197
Green=123,184,123
BoldGreen=197,224,197
Yellow=184,184,123
BoldYellow=224,224,197
Blue=123,123,184
BoldBlue=197,197,224
Magenta=184,123,184
BoldMagenta=224,197,224
Cyan=123,184,184
BoldCyan=197,224,224
White=217,217,217
BoldWhite=255,255,255
```

**Comment by mintty** [comment]
> there is no wsltty in %APPDATA% by default.

Since 0.7.7, %APPDATA%\wsltty should be created during installation.
Can you reproduce a fresh installation and verify it's not being created?

> The current file has the output of the https://ciembor.github.io/4bit/ site copied and pasted into it.

That's not the intended way although it should be working, provided you don't have DOS lineends in your file. The easier way is to drag the output over to the Options menu's Theme field, then give it a name and click Store. So you would get a reference stored in the config file, not the full scheme itself, which is more flexible for later switching.

**Comment by sharkyP** [comment]
I'm having the same issue, the themes are not loading, and the color doesn't change by adding it to the config file

**Comment by mintty** [comment]
Did you consider my previous comment? Please provide your config file (upload) and maybe other individual details of your configuration.

**Comment by sharkyP** [comment]
Hi There
I have tried this - it simply adds the path to the file in Theme field however the store button is in active. I have also added the files in %APPDATA%\wsltty\themes and they appear in the drop down menu and save the ThemeFile entry in the config however it is not applied when I click save or apply

My config is

`
# To use common configuration in %APPDATA%\mintty, simply remove this file
BoldAsFont=no
Term=xterm-256color
BellType=0
FontHeight=11
Columns=120
Rows=30
CursorType=block
ClickTargetMod=alt
AltFnShortcuts=yes
ComposeKey=off
ZoomShortcuts=no
CtrlAltIsAltGr=no
Font=DejaVu Sans Mono for Powerline
CursorBlinks=yes
Scrollbar=none
Language=@
FontSmoothing=full
BoldAsColour=yes
OpaqueWhenFocused=yes
Transparency=medium
CtrlShiftShortcuts=no
ForegroundColour=147,161,161
BackgroundColour=0,43,54
CursorColour=253,157,79
Black=0,43,54
BoldBlack=101,123,131
Red=220,50,47
BoldRed=232,115,113
Green=133,153,0
BoldGreen=199,228,0
Yellow=181,137,0
BoldYellow=255,193,2
Blue=38,139,210
BoldBlue=99,173,227
Magenta=108,113,196
BoldMagenta=162,165,217
Cyan=42,161,152
BoldCyan=71,207,197
White=147,161,161
BoldWhite=253,246,227
ThemeFile=solarized.minttyrc
`

**Comment by mintty** [comment]
> Please provide your config file (upload).

Please *upload* your config file. I have the suspicion it contains DOS line ends (CRLF) in which case the theme name would not match.

**Comment by sharkyP** [comment]
yep that fixed it! thanks :)

**Comment by mintty** [comment]
Released 0.7.8.
